playbooks/synchronise_ssh_keys.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: all
+  tasks:
+    - name: Clone the credential repo
```

Comment:
I think there will be multiple scenarios where we'll need to clone the credential repo - tower bootstrap, integreatly bootstrap, cluster bootstrap etc.

Perhaps we could move this block into a more generic playbook that everything else can call from

Reply:
Yep good idea